.travis.yml

```diff
@@ -4,8 +4,8 @@ sudo: false # faster builds
 
 jdk:
   - openjdk8
-#  - openjdk11 # todo Blocked by https://github.com/kaitai-io/kaitai_struct/issues/483
-#  - oraclejdk11 # todo Blocked by https://github.com/kaitai-io/kaitai_struct/issues/483
+  - openjdk11
```

Comment:
Enabling of JDK 11 led to errors in tests - do you want to fix them?

Reply:
Fixed by using the new parameter for all integration tests